Client/Frontend/Settings/WebsiteDataManagementViewController.swift

```diff
@@ -44,6 +49,7 @@ class WebsiteDataManagementViewController: ThemedTableViewController, UISearchBa
         }
         definesPresentationContext = true
 
+        tableView.isEditing = true
```

Comment:
`tableView.allowsSelectionDuringEditing = true`

Reply:
This isn't needed to enable button taps.